src/amo/components/AddonMoreInfo/index.js

```diff
@@ -173,6 +181,23 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
           {i18n.gettext('Read the license agreement for this add-on')}
         </Link>
       ) : null,
+      relatedCategories: addon.categories && addon.categories[clientApp].length > 0 ? (
+        <ul className="Categories-list Categories-list--justify-content">
+          {addon.categories[clientApp].map((category, index) => {
+              return (
+                <li className="Categories-item" key={category}>
+                  <Button
+                    className={`Categories-link
+                      Categories--category-color-${(index % 12) + 1}`}
+                    to={categoryResultsLinkTo({ addonType, category })}
```

Comment:
I think we can simplify here as we only need the category names without buttons or colors, see the mockup in this comment: https://github.com/mozilla/addons-frontend/issues/2758#issuecomment-339620743

Reply:
Ohk @wildurand. And what about the categories names as only slug is coming ?

Reply:
Ohk @willdurand 👍